Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "robyn"
-version = "0.26.0"
+version = "0.26.000001"
```

Comment:
We don't need to do this anymore. Is this instruction still present somewhere?

Reply:
Oh, I didn't know. Yes, that is the instructions in the PR request.

Reply:
oh, we should fix it then. :D

Also, rustfmt is failing one last time. We can merge it after that is addressed 😄